examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -526,15 +537,28 @@ def compute_loss(params):
             return loss
 
         grad_fn = jax.value_and_grad(compute_loss)
-        loss, grad = grad_fn(state.params)
-        grad = jax.lax.pmean(grad, "batch")
+        loss, grads = grad_fn(state.params)
+        grad_accum = jax.tree_multimap(lambda x, y: x + y, grads, state.grad_accum)
+
+        def update_fn():
```

Comment:
We need better names here and more comments IMO or the script start becoming a bit unreadable